### PR TITLE
Builer with required fields for PreparedDeleteObject

### PR DESCRIPTION
**Pull request by nikitin-da**
@artem-zinnatullin Check, please.

### CODE REVIEW

**Comment by artem-zinnatullin** [comment]
Ffffailed!

**Comment by nikitin-da** [comment]
Noo, it's success =)

File: storio-contentresolver/src/main/java/com/pushtorefresh/storio/contentresolver/operation/delete/PreparedDeleteObject.java [hunk 1]
```diff
@@ -113,8 +115,36 @@ public Builder(@NonNull StorIOContentResolver storIOContentResolver, @NonNull T
...
+
+        CompleteBuilder(@NonNull final Builder<T> builder) {
+            super(builder.storIOContentResolver, builder.object);
+            deleteResolver = builder.deleteResolver;
+            mapFunc = builder.mapFunc;
+        }
+
+        @NonNull
+        @Override
+        public CompleteBuilder<T> withDeleteResolver(@NonNull DeleteResolver deleteResolver) {
```
Comment (artem-zinnatullin) [inline storio-contentresolver/src/main/java/com/pushtorefresh/storio/contentresolver/operation/delete/PreparedDeleteObject.java]: Please use `@inheritdoc` to inherit javadoc from parent class

 ```java
/**
* {@inheritDoc}
*/
```

File: storio-contentresolver/src/main/java/com/pushtorefresh/storio/contentresolver/operation/delete/PreparedDeleteObject.java [hunk 2]
```diff
@@ -113,8 +115,36 @@ public Builder(@NonNull StorIOContentResolver storIOContentResolver, @NonNull T
...
+            super.withDeleteResolver(deleteResolver);
+            return this;
+        }
+
+        @NonNull
+        @Override
+        public CompleteBuilder<T> withMapFunc(@NonNull MapFunc<T, DeleteQuery> mapFunc) {
```
Comment (artem-zinnatullin) [inline storio-contentresolver/src/main/java/com/pushtorefresh/storio/contentresolver/operation/delete/PreparedDeleteObject.java]: Same as above

**Comment by artem-zinnatullin** [comment]
Magic! :)

**Comment by nikitin-da** [comment]
ok

**Comment by artem-zinnatullin** [comment]
Thanks!

**Comment by artem-zinnatullin** [comment]
Refs #218

**Pull request merged**